handsontable/src/pluginHooks.js

```diff
@@ -2911,8 +2911,14 @@ class Hooks {
   createEmptyBucket() {
     const bucket = Object.create(null);
 
+    bucket._order = Object.create(null);
```

Comment:
If you try to use the hook name under the same name, you will get an error `hot.addHook('_order', () => {})`. Did you consider using `WeakMap`'s here?